### PR TITLE
Create House-of-Permaculture.md

### PR DESCRIPTION
**Pull request by Uranus5007**
Create initial File

### CODE REVIEW

File: docs/great-houses/House-of-Permaculture.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<center><img src="https://cdn.discordapp.com/attachments/788121740256608276/808398234778337280/PermacultureHeaderInvert.png" alt="The Metagame House of Permaculture" width="760" height="324"></center><p class="big">
```
Comment (hammadj) [inline docs/great-houses/House-of-Permaculture.md:1]: Looks like this image tag is missing a closing tag, need to fix the syntax

**Comment by Uranus5007** [comment]
Ready to submit PR Please x

**Comment by dysbulic** [comment]
The format of the HTML produced by StackEdit doesn't conform to the standard used by the wiki. Someone is going to have to go over it and fix the tags or convert it to Markdown.

**Comment by Uranus5007** [comment]
I'll do it tomorrow

**Pull request closed without merging**